specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -78,3 +78,94 @@ model Evaluation {
   @doc("Evaluators to be used for the evaluation.")
   evaluators: Record<EvaluatorConfiguration>;
 }
+
+@doc("Definition for sampling strategy.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v_latest)
+model SamplingConfiguration {
+  @doc("Name of the sampling strategy.")
+  name: string;
+
+  @doc("Percentage of sampling per hour (0-100).")
+  samplingPercent: float;
+
+  @doc("Maximum request rate per hour (0 to 1000).")
+  maxRequestRate: float;
+}
+
+@doc("The redaction configuration will allow the user to control what is redacted.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v_latest)
+model RedactionConfiguration {
```

Comment:
This also seems specific to output configuration. Maybe we want to add OutputConfiguration instead?

Reply:
Renamed `RedactionConfiguration` to `AgentEvaluationRedactionConfiguration`. Let's discuss offline for the naming

Reply:
Discussed offline, we will cover this by the evaluation API convergence project